content/pt-br/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -0,0 +1,104 @@
+escalar e excluir os membros do StatefulSet. Ele tenta garantir que o número especificado de Pods,
+do ordinal 0 até N-1, esteja ativo e pronto. O StatefulSet garante que, a qualquer momento,
+exista no máximo um Pod com uma determinada identidade em execução no cluster. Isso é chamado de semântica
+*no máximo um* fornecida por um StatefulSet.
+
+A exclusão forçada manual deve ser realizada com cautela, pois tem o potencial de violar a semântica de *no máximo um*
+inerente ao StatefulSet. StatefulSets podem ser usados para executar aplicações distribuídas e em cluster que
+necessitam de uma identidade de rede estável e armazenamento estável. Essas aplicações frequentemente possuem
+configurações que dependem de um conjunto fixo de membros com identidades fixas. Ter múltiplos membros com a mesma
+identidade pode ser desastroso e pode levar à perda de dados (por exemplo, cenário de split brain em sistemas baseados em quórum).
+
+## Excluir Pods
+
+Você pode realizar uma exclusão graciosa de um Pod com o seguinte comando:
+
+```shell
+kubectl delete pods <pod>
+```
+
+Para que o procedimento acima resulte em uma finalização graciosa, o Pod **não deve** especificar um
+`pod.Spec.TerminationGracePeriodSeconds` igual a 0. A prática de definir `pod.Spec.TerminationGracePeriodSeconds`
+como 0 segundos é insegura e fortemente desaconselhada para Pods de StatefulSet.
+A exclusão graciosa é segura e garantirá que o Pod
+[seja finalizado de forma adequada](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination) antes que o
+kubelet remova o nome do Pod do servidor de API.
+
+Um Pod não é excluído automaticamente quando um nó se torna inacessível. Os Pods em execução em um Nó
+inacessível entram no estado 'Terminating' ou 'Unknown' após um [timeout](/docs/concepts/architecture/nodes/#condition).
+Os Pods também podem entrar nesses estados quando o usuário tenta realizar a exclusão graciosa de um Pod em um Nó inacessível.
+As únicas formas de remover um Pod nesse estado do servidor de API são as seguintes:
+
+- O objeto Node é excluído (por você ou pelo [Node Controller](/docs/concepts/architecture/nodes/#node-controller)).
+- O kubelet no Nó sem resposta volta a responder, encerra o Pod e remove a entrada do servidor de API.
+- Exclusão forçada do Pod pelo usuário.
+
+A prática recomendada é utilizar a primeira ou a segunda abordagem. Se um Nó for confirmado como morto
+(por exemplo, desconectado permanentemente da rede, desligado, etc.), exclua o objeto Node.
+Se o Nó estiver sofrendo uma partição de rede, tente resolver o problema ou aguarde até que ele seja resolvido.
+Quando a partição for sanada, o kubelet concluirá a exclusão do Pod e liberará seu nome no servidor de API.
+
+Normalmente, o sistema conclui a exclusão assim que o Pod não está mais em execução
+em um Nó ou quando o Nó é excluído por um administrador.
+Você pode substituir esse comportamento forçando a exclusão do Pod.
```

Comment:
Esse parágrafo está misturando os termos Nó e Node. A página original se refere à API. No geral, na documentação em português temos ambas as formas. Não tenho uma preferência muito forte por um ou por outro, mas temos que manter o termo constante ao longo da página.

@edsoncelio tens alguma preferência com relação ao uso de Nó ou Node?

Reply:
Vou contribuir aqui, sempre preferi em português os termos, porém quando são termos mais comuns, como é o caso deste, poderia ser assim no início Nó (Node), ou seja, na primeira menção ao termo e depois continuar apenas com Nó.

Parabéns  pela dedição pessoal!